README.Rmd

```diff
@@ -85,7 +85,7 @@ ggthemr('flat dark', spacing = 0.5, type = 'inner')
 preview_theme()
 ```
 
-### camoflauge
+### camouflage
```

Comment:
See #39 -- `palettes.R` needs to be changed too?

Reply:
👍